groups/api/http/endpoints.go

```diff
@@ -180,14 +180,16 @@ func ListGroupsEndpoint(svc groups.Service) endpoint.Endpoint {
 			groups = append(groups, toViewGroupRes(g))
 		}
 
-		return groupPageRes{
+		res := groupPageRes{
 			pageRes: pageRes{
 				Limit:  page.Limit,
 				Offset: page.Offset,
 				Total:  page.Total,
 			},
 			Groups: groups,
-		}, nil
+		}
+
+		return res, nil
 	}
```

Comment:
Please revert back to these lines to `main` branch version
```suggestion
		return groupPageRes{
			pageRes: pageRes{
				Limit:  page.Limit,
				Offset: page.Offset,
				Total:  page.Total,
			},
			Groups: groups,
		}, nil
	}
```